core/domain/html_cleaner.py

```diff
@@ -378,11 +378,13 @@ def validate_rte_tags(
             'SkillReview'
         )
 
-        _raise_validation_errors_for_unescaped_html_tag(
-            tag,
-            'skill_id-with-value',
-            'SkillReview'
-        )
+        # TODO(#16820): Investigate why skill_id attribute empty error occured
```

Comment:
Never comment out code. Delete it and reinstate it later if needed (but still have a TODO here).

That said, I'm OK with waiting a couple days if it means getting to the actual root of the issue, otherwise we're just creating extra future work. Please could you do a bit more investigation to see why this error is happening so that we can fix it at its root, once and for all?